mw-webapp/src/component/header/Header.tsx

```diff
@@ -19,8 +22,9 @@ const LOGO_TEXT = "master's way";
 export const Header = () => {
   const {user} = useGlobalContext();
   const navigate = useNavigate();
+  const {theme, onChangeTheme} = useChangeTheme();
```

Comment:
Are you sure then we need to use theme in the header? Now it's on settings page ))

Reply:
It's pretty nice demonstration!) Thanks! I think we are going to use sun and moon icons to handle theme in near future instead of dropdown. As I know we have actions on the the way page (delete way, download as pdf), we can use dropdown with them. Or just skip it for now and implement it in another task. It's up to you.

Reply:
done